src/components/markets/MarketsListItem.tsx

```diff
@@ -25,10 +26,15 @@ export const MarketsListItem: React.FC<MarketsListItemProps> = ({ reserveData, s
         marginBottom: '12px',
         borderRadius: '5px',
         '&:hover': {
+          cursor: 'pointer',
```

Comment:
i don't think we do this consistently :(

gonna make an issue to do so

Reply:
assign me to it once you do